src/Main/Views/Expenses/Table/Table.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+import { expensesData } from "../data/expensesData.js";
+import TableRow from "./TableRow";
+
+import "./Table.css";
+
+const Table = () => {
+
+    const data = expensesData;
+    data.forEach(entry => {
+        return entry.date = entry.date.slice(0, 10);
+    });
+
+    return (
+        <table className="Table">
+            <tbody>
+                <tr className="Table-column-title">
+                    <td>Date</td>
+                    <td>Payee</td>
+                    <td>Category</td>
+                    <td>Memo</td>
+                    <td>Outflow</td>
+                    <td>Inflow</td>
+                </tr>
+                <TableRow date={data[0].date}
```

Comment:
Na spotkaniu pogadamy jeszcze o bardziej dynamicznym tworzeniu takich struktur :) Tak na teraz podrzucam: https://pl.reactjs.org/docs/thinking-in-react.html